infra/cifuzz/cifuzz.py

```diff
@@ -186,6 +196,14 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   if not os.path.exists(workspace):
     logging.error('Invalid workspace: %s.', workspace)
     return False, False
+
+  # Check that sanitizer is valid.
+  if not is_project_sanitizer(sanitizer, project_name):
+    logging.info("%s is not a project sanitizer, defaulting to address.",
```

Comment:
Generally it's better to break loudly instead. How will users know they made a typo if this message is hidden inside long logs and everything is green?

also use single quotes for strings.

Reply:
+1 definitely better to break in this case.

Reply:
Good point. I will raise an error.